source/edge-server/manage-edge-servers.txt

```diff
@@ -460,6 +481,13 @@ will have the option to recover local data when this error occurs.
 You can change the length of the client maximum offline time in the 
 :ref:`sync-settings`.
 
+.. important:: Edge Server Offline Limit is 60 Days
+
+   An Edge Server instance cannot be offline longer than 60 days, no matter
+   what setting you set for the client maximum offline time. If the setting
+   is ``nil`` or greater than 60 days, the Edge Server offline limit is
```

Comment:
Oh we should say "If the setting is unset" I think instead of nil here. My bad, I was using Golang terminology here.